neonvm/tools/vm-builder/files/vminit

```diff
@@ -50,3 +50,11 @@ resize2fs /dev/vda
 # networking
 ip link set up dev lo
 ip link set up dev eth0
+
+# ssh
+/neonvm/bin/ssh-keygen -A
```

Comment:
How long does this take? From the docs of `-A`:

> Generate host keys of all default key types (rsa, ecdsa, and ed25519) if they do not already exist

Maybe we should only use ed25519 for efficiency?

Reply:
Openssh is quite fast, this takes ~15ms. I'll try to see how it works with only one pair of keys.

Reply:
Hm... when trying this out in a debian container locally, I see that on install, it generates keys, but manually specifying a different path to generate them in causes it to take ~0.5s:

```
$ docker run -it debian:latest bash
root@6eb1acea5942:/# apt update && apt install -y openssh-server
... output omitted ...
root@6eb1acea5942:/# mkdir -p foo/etc/ssh
root@6eb1acea5942:/# time ssh-keygen -A -f foo
ssh-keygen: generating new host keys: RSA ECDSA ED25519 

real	0m0.659s
user	0m0.655s
sys	0m0.004s
```

(got varied results on re-runs)

Reply:
Anyway it's better to only use `ed25519` keys since the rest would be useless. Resolved here 4c4e56a6b9cd41a4c50a5ca8662dcee8e265d4e1